features/step_definitions/web_steps.rb

```diff
@@ -20,6 +20,24 @@ def with_scope(locator)
   visit path_to(page_name)
 end
 
+When(/^I visit "([^"]*)" it should fail with an error$/) do |path|
+  expect {
+    visit path
+  }.to raise_error
+end
+
+When /^I logout using the browser$/ do
+  step %{I follow "Log Out"}
+end
+
+When /^I limit myself to the Archive$/ do
```

Comment:
It is setting it so that the phantomjs client doesn't pull things from all over the internet. Originally it was because the relevant paths were going to example.com . However it makes the debug output a lot easier to read. 

While I can take it out of the feature now, it is still a useful step.